### DISCUSSION TITLE
Substack

### DISCUSSION THREAD

**Discussion by nekowinston** [catppuccin/catppuccin#2073] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Userstyle: Stylus

### Any additional comments?

https://github.com/nekowinston/catppuccin-substack/blob/main/catppuccin.user.css

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
@isabelroses Are you interested in taking winston's code over to our new userstyles repository?

**Reply by nekowinston** [reply]
I'd like to further refine the code myself now that userstyles has been transferred, certainly isn't high priority to get moved over as well.

**Comment by nekowinston** [comment]
Will be handled over at catppuccin/userstyles#113
